src/Dialect/ONNX/Rewrite.cpp

```diff
@@ -18,23 +18,17 @@
 
 #include "src/Dialect/ONNX/ONNXOps.hpp"
 #include "src/Dialect/ONNX/ONNXOpsHelper.hpp"
+#include <numeric>
```

Comment:
Why do we need to include numeric?

Reply:
yes now that I reverted code that used std::accumulate we do not need it anymore, will remove.